gradle.properties

```diff
@@ -1,6 +1,6 @@
 # modify this file to change project properties
```

Comment:
This sets default configuration for playtest command. So, it runs our bot vs the "oldbot"